include/deal.II/numerics/vector_tools.templates.h

```diff
@@ -6373,6 +6374,11 @@ namespace VectorTools
       dealii::hp::FECollection<dim,spacedim> fe_collection (dof.get_fe());
       IDScratchData<dim,spacedim> data(mapping, fe_collection, q, update_flags);
 
+      //FIXME
+      // temporary vectors of consistent with InVector type
+      std::vector<dealii::Vector<Number>> function_values;
+      std::vector<std::vector<Tensor<1,spacedim,Number> >> function_grads;
```

Comment:
You need these because `IDScratchData` does not have a template type for the `InVector`, right? If so, please leave a note with the FIXME explaining why and we get to it in a separate patch later on (unless you just want to add another template argument to `IDScratchData` that either denotes the vector type, or the underlying scalar type.